### ISSUE TITLE
Ubuntu 13.10 redis-desktop-manager_0.7.5_i386_ubuntu.deb

### ISSUE THREAD

**Issue by ghost** [redis/RedisDesktopManager#571]
rmd fails to start with unresolved libs

ldd rdm
        linux-gate.so.1 =>  (0xf7780000)
        libssh2.so.1 => /usr/share/redis-desktop-manager/bin/./../lib/libssh2.so.1 (0xf7754000)
        libQt5Widgets.so.5 => /usr/share/redis-desktop-manager/bin/./../lib/libQt5Widgets.so.5 (0xf7153000)
        libQt5Xml.so.5 => /usr/share/redis-desktop-manager/bin/./../lib/libQt5Xml.so.5 (0xf7118000)
        libQt5Network.so.5 => /usr/share/redis-desktop-manager/bin/./../lib/libQt5Network.so.5 (0xf6fce000)
        libQt5Gui.so.5 => /usr/share/redis-desktop-manager/bin/./../lib/libQt5Gui.so.5 (0xf6c08000)
        libQt5Core.so.5 => /usr/share/redis-desktop-manager/bin/./../lib/libQt5Core.so.5 (0xf6720000)
        libpthread.so.0 => /lib/i386-linux-gnu/libpthread.so.0 (0xf66e8000)
        libstdc++.so.6 => not found
        libgcc_s.so.1 => /lib/i386-linux-gnu/libgcc_s.so.1 (0xf66ca000)
        libc.so.6 => /lib/i386-linux-gnu/libc.so.6 (0xf6516000)
        libcrypto.so.1.0.0 => /lib/i386-linux-gnu/libcrypto.so.1.0.0 (0xf6369000)
        libz.so.1 => /lib/i386-linux-gnu/libz.so.1 (0xf634f000)
        libX11.so.6 => not found
        libstdc++.so.6 => not found
        libm.so.6 => /lib/i386-linux-gnu/libm.so.6 (0xf630b000)
        libstdc++.so.6 => not found
        libssl.so.1.0.0 => /lib/i386-linux-gnu/libssl.so.1.0.0 (0xf62b4000)
        libstdc++.so.6 => not found
        libstdc++.so.6 => not found
        libdl.so.2 => /lib/i386-linux-gnu/libdl.so.2 (0xf62ae000)
        librt.so.1 => /lib/i386-linux-gnu/librt.so.1 (0xf62a5000)
        /lib/ld-linux.so.2 (0xf7781000)
        libglib-2.0.so.0 => not found
        libstdc++.so.6 => not found

**Comment by chymian** [comment]
a amd64 version is needed - or a proper how-to install since ubuntu Bug #1123034 states: 
https://bugs.launchpad.net/ubuntu/+source/dpkg/+bug/1123034/comments/2
Ubuntu 12.04 has an earlier implementation of multiarch where it uses the --foreign-architecture options that you see in /etc/dpkg/dpkg.cfg.d/multiarch. Newer versions of ubuntu have the latest multiarch implementation where dpkg --add-architecture is supported.

This is not a bug. I don't know what you expect. In any case, the new multiarch implementation won't be deployed to LTS 12.04 at this point.

there is nothing in /usr/share/doc/redis-desktop-manager/README

and rdm should support 12.04LTS – from the download page:
Supported Ubuntu versions: 12, 13

this is a severe bug! it's not working!

EDIT:
sorry, I overlooked that this was opened against ubuntu 13.10.
but it's true for 12.04LTS as well, as I described, even more problematic.
redis-desktop-manager:i386           0.7.5
